tests/test_routes.py

```diff
@@ -43,3 +58,44 @@ def test_index(self):
         """ It should call the home page """
         resp = self.client.get("/")
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
+
+    def test_create_inventory_success(self):
+        """It should Create a new Inventory item"""
+        test_inventory = InventoryFactory()
+        logging.debug("Test Inventory create successful: %s", test_inventory.serialize())
+        response = self.client.post(BASE_URL, json=test_inventory.serialize())
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        # Make sure location header is set
+        location = response.headers.get("Location", None)
```

Comment:
What's the location in the header?

Reply:
Create endpoints are supposed to redirect after successful creation but I have to redirect to the List endpoint which doesn't actually exist yet so I'll just make a separate ticket for that in the near future.
Removing it for now! Good catch.